src/hfsfuse.c

```diff
@@ -48,6 +48,7 @@ struct hf_file {
 	hfs_extent_descriptor_t* extents;
 	uint16_t nextents;
 	uint8_t fork;
+	uint8_t is_empty;
```

Comment:
Even though these are almost certainly typedef'd to the same thing, can you add stdbool to the other stdlib includes in this file and use the actual `bool` type for this?
We target C11 so don't hesitate to use newer stuff in the codebase as long as it's not an extension.

Reply:
Sure!—done and squashed.

Reply:
Thank you!